eng/pipelines/templates/stages/archetype-cpp-release.yml

```diff
@@ -1,3 +1,5 @@
+#cspell: words gearama
```

Comment:
We shouldn't need this anymore, correct? Because it is in the global list now?

Reply:
Thanks! Missed that one while updating. Fixed now.